doc/specs/#194 - PWA Support.md

```diff
@@ -0,0 +1,123 @@
+
+```
+Id: Microsoft.FinTimes
+Version: 1.0.0.0
+Name: AppInstaller FinTimes
+Publisher: Microsoft Corporation
+AppMoniker: fintimes
+License: Test
+InstallerType: PWA
+MinOSVersion: 10.0.0.0
+Installers:
+- Arch: x64
+  Url: https://app.ft.com
+  InstallerType: PWA
+  ManifestVersion: 0.1.0
+```
+
+### Package generation
+
+A POST call is made to a web service (https://pwabuilder-win-chromium-platform.centralus.cloudapp.azure.com) with a JSON body containing the URL to the PWA. The web service packages the PWA and its assets into an unsigned MSIX. An example of the JSON body would be :
+
+```
+{
+    "url": "https://app.ft.com"
+}
+```
+
+There are many more optional parameters that can be passed to this service. For more information, refer to the test website linked in the Resources section.
+
+### Package installation
+
+The unsigned MSIX package and any other dependency packages are added for the current user and the application is silently installed on the user's system. The installation is done by calling the Windows API AddPackageByUriAsync() using the specific deployment options that allow unsigned packages.
+
+### Flow of the install process
+
+1. User types the install command, eg. `winget install fintimes` (For the experimental version, the manifest will be passed as an argument)
+2. The URL from the manifest YAML file of the Financial Times PWA will be sent as a POST request to an internal web service that will generate an unsigned MSIX package. This MSIX package is downloaded onto the user's system.
+3. The generated package will then be silently installed as a Hosted App (Refer to resources for more on the Hostel App Model).
+4. The MSIX file is removed from the user's system.
+
+## UI/UX Design
+
+Installing a PWA will be similar to installing other application types currently supported by winget, using the install command. The only difference in experience might be if the user passes a -i flag to get a more interactive experience. When this flag is passed, the user will be prompted to allow the PWA to be launched on install. This feature is provided because in order to register a PWA with Edge, it has to be launched once.
+
+An example of the interactive install would look like this:
+
+```
+>winget install -i fintimes
+Found Financial Times [PWAtest.FinTimes]
+This application is licensed to you by its owner.
+Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
+Starting package install...
+<Progress bar>
+Succesfully installed! Launching app now. This is a necessary step to complete installation.
+(App launches on Edge)
+```
+
+## Capabilities
+
+### Accessibility
+
+This should have no direct impact on accessibility.
+
+### Security
+
+No direct impact on security.
+
+### Reliability
+
+This is not expected to impact reliability. It will be available as an experimental feature so users can disable it any time.

```

Comment:
The spec is not for the experimental feature, it should be for the released one.  The experimental stage is along the path.

Reply:
Okay, I removed that sentence on `[this](https://github.com/microsoft/winget-cli/pull/563/commits/5b642f55005a3d5bcede69322e08a654f733ab96)`  commit. Had to rename the file there was a mismatch in the issue number.